dj_hetmech_app/views.py

```diff
@@ -177,6 +192,12 @@ def get(self, request):
 
 
 class QueryPathsView(APIView):
+    """
+    For a given source node, target node, and metapath, return the actual paths comprising the path count / DWPC.
+    These paths have not been pre-computed and are extracted on-the-fly from the Hetionet Neo4j Browser.
+    Therefore, it is advisable to avoid querying a source-target-metapath pair with a path count exceeding 10,000.
+    Because results are ordered by PDP / percent_of_DWPC, reducing max_paths does not prevent neo4j from having to exhaustively traverse all paths.
```

Comment:
Isn't this line too long?

Reply:
Since this docstring is getting parsed as markdown, I used the one sentence per line construct we usually use for markdown prose. I will go through other docstrings so that this is consistent.